client/src/components/CompanyRegister.js

```diff
@@ -0,0 +1,194 @@
+                  Create New Company
+                </Header>
+              </Grid.Column>
+            </Grid.Row>
+            <Grid.Row centered>
+              <Grid.Column>
+                <Form.Field
+                  label="Company Name"
+                  control={Input}
+                  placeholder="Company Name"
+                  required
+                  value={this.state.companyProfile.name}
+                  name="name"
+                  onChange={this.handleChange}
+                >
+                  <input />
+                </Form.Field>
+                <Form.Field
+                  control={Input}
+                  label="Email"
+                  placeholder="Email"
+                  name="email"
+                  type="email"
+                  value={this.state.companyProfile.email}
+                  required
+                  onChange={this.handleChange}
+                >
+                  <input />
+                </Form.Field>
+                <Form.Field
+                  label="Password"
+                  control={Input}
+                  type="password"
+                  placeholder="Password"
+                  name="password"
+                  value={this.state.companyProfile.password}
+                  required
+                  onChange={this.handleChange}
+                >
+                  <input />
+                </Form.Field>
+                <Form.Field
+                  floated="left"
+                  label="Company Description"
+                  control={TextArea}
+                  placeholder="Company Description"
+                  name="description"
+                  value={this.state.companyProfile.description}
+                  required
+                  onChange={this.handleChange}
+                ></Form.Field>
+                <Form.Field>
+                  <label> Select Industry</label>
+                  <Dropdown
+                    label="Industry"
+                    options={industryList}
+                    search
+                    selection
+                    value={this.state.companyProfile.industry}
+                    required
+                    placeholder="Select Industry"
+                    onChange={this.handleIndustry}
+                  />
+                </Form.Field>
+                <Button
+                  type="submit"
+                  primary
+                  fluid
+                  // onClick={this.state.success === true ? this.resetForm : null}
+                >
+                  Submit
+                </Button>
+              </Grid.Column>
+            </Grid.Row>
+            <Grid.Row width={15}>
+              {this.state.success === true && (
+                <Message positive>
+                  <Message.Header>
+                    Company Profile Created successfully
+                  </Message.Header>
+                </Message>
+              )}
+              {this.state.success === false && (
```

Comment:
Do you need to check if there is an error as well?
so you should check if success is false or if the error is true